src/Bpi/ApiBundle/Domain/Entity/ChannelFacet.php

```diff
@@ -0,0 +1,89 @@
+<?php
+namespace Bpi\ApiBundle\Domain\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * Bpi\ApiBundle\Domain\Entity\ChannelFacet
+ */
+class ChannelFacet
+{
+    /**
+     * @var MongoId $id
+     */
+    protected $id;
+
+    /**
+     * @var string $channelId
+     */
+    protected $channelId;
+
+    /**
+     * @var collection $facetData
+     */
+    protected $facetData;
+
+    /**
+     * Create a new ChannelFacet.
+     */
+    public function __construct()
+    {
+        $this->facetData = new ArrayCollection();
```

Comment:
In ChannelFacetFepository class to facetData parameter casted into stdObject with (object), but here it's declared as ArrayColection.